components/desktop/partials/KnitProjectsWidgetItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="knit-projects-widget-item">
+    <div class="knit-projects-widget-item__top-section">
+      <img :src="projectLogoSource || 'images/teal-star.png'" alt="" class="knit-projects-widget-item__project-logo">
+      <h3 class="knit-projects-widget-item__title"> {{ title }} </h3>
+    </div>
+    <div class="knit-projects-widget-item__bottom-section">
+      <ul class="knit-projects-widget-item__list">
+        <li v-if="isNotEmpty(tags)" class="knit-projects-widget-item__list-item">
+          <span class="knit-projects-widget-item__list-item-logo flaticon-curly-brackets"/>
+            {{ arrayToStringList(tags) }}
+        </li>
+        <li v-if="maxTeamSize" class="knit-projects-widget-item__list-item">
+          <span class="knit-projects-widget-item__list-item-logo flaticon-user"/>
+            Max: {{ maxTeamSize }}, Aktualnie: {{ !currentTeamSize ? 0 : currentTeamSize }}
+        </li>
+        <li v-if="isNotEmpty(repoLink)" class="knit-projects-widget-item__list-item">
+          <span class="knit-projects-widget-item__list-item-logo flaticon-github-logo"/>
+            <a :href="repoLink" class="knit-projects-widget-item__list-item-link">
+              {{ trimString(repoLink, 50) }}
+            </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+import templateHelper from '~/helpers/templateHelper.js'
+
+export default {
+  data () {
+    return {}
+  },
+  props: [
+    'tags',
+    'title',
+    'repoLink',
+    'maxTeamSize',
+    'currentTeamSize',
+    'projectLogoSource'
+  ],
+  computed: {},
+  mixins: [ templateHelper ]
+}
+</script>
+
+<style lang="scss">
+@import "assets/scss/imports.scss";
+.knit-projects-widget-item {
+  border-top: 1px solid $knit-projects-widget-item-border-color;
+  border-bottom: 1px solid $knit-projects-widget-item-border-color;
+  padding-bottom: 15px;
+  &:hover {
+    background-color: $knit-projects-widget-item-hover-bg-color;
+    cursor: pointer;
+  }
+  &__top-section {
+    display: flex;
+    padding: 15px 0 10px 15px;
+  }
+  &__title {
+    align-self: center;
+    font-weight: normal;
+    margin-left: 15px;
+  }
+  &__list {
+    font-size: .75rem;
```

Comment:
What about our sass units? I remember, we decided to not use `rems`, because they're problematic to calculate. Am I right?

Reply:
As far as I remember you convinced us that we should use `rems` for fonts

Reply:
Oh, you're right. So I have to change it in my styles :)